src/LogCleaner.cc

```diff
@@ -759,19 +768,19 @@ LogCleaner::Balancer::isMemoryLow(CleanerThreadState* thread)
     const int T = cleaner->segmentManager.getMemoryUtilization();
     const int L = cleaner->cleanableSegments.getLiveObjectUtilization();
 
+    // TODO: When we exceed dangerThreshold, we should probably
+    // do something, but for now, we warn the user
+    if (T >= cleaner->dangerThreshold) {
+        LOG(WARNING, "Memory Utilization (%d) exceeds Danger Threshold", T);
```

Comment:
How often is this checked, and potentially logging?  Is it another log flooder?

Reply:
It isn't, at least not so far, based on the runs Dan & I are doing.